scripts/copy-wellknown.cjs

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env node
+
+/* eslint-disable @typescript-eslint/no-var-requires */
+const path = require('path');
+const fs = require('fs');
+
+const wellKnownPath = path.join(__dirname, '../.well-known');
+
+console.log('Copying .well-known/ contents to apps');
+fs.cpSync(wellKnownPath, path.join(__dirname, '../apps/web/public/.well-known'), {
```

Comment:
`cpSync` is in Stage 1 and it says the following on their page *Use of the feature is not recommended in production environments.*

Should we go ahead and use it?

Reply:
@Mythie unsure about this. other than that, this is good to go IMO

Reply:
It's a script rather than something in production it's fine.